README.md

```diff
@@ -36,7 +32,7 @@ Under this component the implementation of each major version will have a unique
 
 #### Implement a new API with a new swagger definition - version one
 
-1. Include the API swagger definition in the given location of this maven project (identity-api-server). Suggested name 
+1. Convert the API swagger definition to OpenAPI 3.0 and include in the given location of this maven project (identity-api-server). Suggested name 
```

Comment:
Include the API swagger definition (in OpenAPI 3.0) in the given location of this maven project (identity-api-server). The suggested name for the file name of the API definition is `<resource>.yaml`. You have to convert the API definition to OpenAPI 3.0 if you are working with Swagger 2.0 still.